assets/js/components/ActivateModuleCTA.js

```diff
@@ -47,22 +44,23 @@ const ActivateModuleCTA = ( { moduleSlug, title, description } ) => {
 
 	const { activateModule } = useDispatch( CORE_MODULES );
 	const { navigateTo } = useDispatch( CORE_LOCATION );
+	const { setInternalServerError } = useDispatch( CORE_SITE );
 
 	const onCTAClick = useCallback( async () => {
 		const { error, response } = await activateModule( moduleSlug );
 
 		if ( ! error ) {
 			navigateTo( response.moduleReauthURL );
 		} else {
-			showErrorNotification( GenericError, {
+			setInternalServerError( {
```

Comment:
@Hazlitte let's move `type`, `format`, and `title` properties to the `setInternalServerError` action and make them as default values to eliminate a need to repeat them in every component.